pkg/machinery/constants/constants.go

```diff
@@ -780,3 +780,12 @@ const (
 	HostnameKey     = "hostname"
 	MacKey          = "mac"
 )
+
+// Overlays is the set of paths to create overlay mounts for.
+var Overlays = []string{
```

Comment:
I hate that this is here. I had to pull it out of the `mount` package since darwin builds are incompatible with that package. Open to other ideas. I did this temporarily (hopefully) to keep things moving.